dcgm/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
```

Comment:
nit - you don't need this as it's py3 only

Reply:
done!